README.md

```diff
@@ -221,7 +221,7 @@ Content-Type: application/json
         "word": String,             // primary correct answer
         "article": Integer,         // ID of article (german grammar) belonging to the item (1:Der, 2:Die, 3:Das, 4:Die (Plural))
         "audio": String,            // URL to (converted) audio file
-        "word_type": String,        // Word type of document: Nomen, Verb, Adjektiv
+        "word_type": Integer,        // Word type of document: Nomen, Verb, Adjektiv
```

Comment:
```suggestion
        "word_type": String,        // Word type of document: Nomen, Verb, Adjektiv
```
The API response should stay the same, since the app probably doesn't know how to handle the integer values yet. This would be a task for a new API version.